dkpro-core-io-brat-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/io/brat/BratWriter.java

```diff
@@ -610,13 +614,40 @@ private void writeRelationAnnotation(BratAnnotationDocument aDoc, FeatureStructu
         }
     }
 
+
+    
+    private BratTextAnnotation splitNewline(AnnotationFS aFS)
+    {
+
+        // extract all but newlines as groups
+        Matcher m = newlineExtractPattern.matcher(aFS.getCoveredText());
+        // counts the number of matches to initialize arrays sized
+        int i = 0;
+        while (m.find()) {
+            i++;
+        }
+        // initialize arrays 
+        int[] begins = new int[i];
+        int[] ends = new int[i];
+        m.reset();
+        i = 0;
+        while (m.find()) {
+            begins[i] = m.start(1) + aFS.getBegin();
+            ends[i] = m.end(1) + aFS.getBegin();
+            i++;
+        }
+        // replaces any group of newline by one space
+        String[] texts = new String[] { aFS.getCoveredText().replaceAll("\\R+", " ") };
```

Comment:
Is this what brat does? I'm not sure if it is a good idea to conflate consecutive newline characters into a single space as that potentially introduces a difference between the text length and the difference between the start/end offsets. I suppose a safer approach would be to replace every single newline character with a space character.

Reply:
Indeed, that's what brat does (at least V1.3 version).